prac_08/silver_service_taxi.py

```diff
@@ -1,20 +0,0 @@
-"""
-CP1404/CP5632 Practical
-SilverServiceTaxi class
-"""
-from prac_08.taxi import Taxi
-
-
-class SilverServiceTaxi(Taxi):
-    flagfall = 4.5
-
```

Comment:
I'm only commencing Prac8 today so ignore if I'm wrong, but this looks like constant value... should it be in caps (FLAGFALL)?